web-server/v0.4/src/models/global.js

```diff
@@ -8,6 +8,8 @@ export default {
     datastoreConfig: {},
     indices: [],
     selectedIndices: [],
+    selectedResults: [],
```

Comment:
Does having selected indices and results in the global redux store mean that it will be shared between all pages that import global?  And if so, how might that affect behaviors between pages?

Reply:
That's correct - `global` is another redux namespace used to track information that is shared globally throughout the dashboard. For example, we use the `selectedResults` state in the Results, ComparisonSelect, and RunComparison page components. This doesn't change the behaviors of pages as the page components are now just pulling this information from a separate key in the redux store.